### PR TITLE
added setFormSubmitProcessor to HandlerConfigInterface

### PR DESCRIPTION
**Pull request by phonixor**
hostnet/form-handler-bundle#39
lets try this again

### CODE REVIEW

File: src/FormHandler/HandlerConfigInterface.php [hunk 1]
```diff
@@ -52,6 +52,18 @@ public function onSuccess(callable $callback);
     public function onFailure(callable $callback);
 
     /**
+     * Register the FormSubmitProcessor
```
Comment (iltar) [inline src/FormHandler/HandlerConfigInterface.php]: Missing a dot on the end, for consistency purposes please add one

File: src/FormHandler/HandlerConfigInterface.php [hunk 2]
```diff
@@ -52,6 +52,18 @@ public function onSuccess(callable $callback);
...
+     *
+     * This will overwrite the default FormSubmitProcessor.
+     *
+     * signature:
+     * function (FormInterface $form, Request $request): void;
+     *
+     * @param callable $callback
+     */
+    public function setFormSubmitProcessor(callable $callback);
```
Comment (iltar) [inline src/FormHandler/HandlerConfigInterface.php]: Don't forget to add a BC notice about this

File: src/upgrade_1.5.md [hunk 3]
```diff
@@ -0,0 +1,2 @@
+BC break:
+- added setFormSubmitProcessor(callable $callback) to HandlerConfigInterface
```
Comment (iltar) [inline src/upgrade_1.5.md]: `UPGRADE-1.5.md`

```
- Added a new method to HandlerConfigInterface:
  `setFormSubmitProcessor(callable $callback)`. While technically a BC break,
  this interface is not designed to be implemented with a custom configuration.
```

File: src/FormHandler/HandlerConfigInterface.php (hunk 2 above)
Comment (phonixor) [reply src/FormHandler/HandlerConfigInterface.php]: done

**Pull request merged**